docs/ru/easytofly.md

```diff
@@ -0,0 +1,89 @@
+
+Далее встал вопрос об использовании датчиков для разработки данной подсистемы.
+
+#### LIDAR
+
+Первое, что приходит в голову при разработке подобной системы – круговые LIDAR-датчики (далее – лидар). Самым популярным похожим решением на сегодняшний день является RPLIDAR A1.
+
+Датчик основан на принципе лазерной триангуляции и использует высокоскоростное оборудование для получения и обработки изображений, система измеряет данные о расстоянии более 8000 раз в секунду. Лидар вращается по часовой стрелке для выполнения всенаправленного лазерного сканирования окружения на 360 градусов.
+
+Результатом является облако точек, которое можно использовать при дальнейшей работе с пакетами ROS, позволяющими производить действия с этой структурой данных (такие как: перевод в другие структуры данных, построение карты, передача данных на другое оборудование).
+
+Для установки датчика на Clover 4 было разработано и подготовлено для печати на 3D принтере специальное крепление, которое для экономии высоты было совмещено с креплением для аккумулятора.
+
+Далее предстояла разработка программного обеспечения, которое выполняло все преобразования, как типов данных, так и геометрии (соглашения о системах координат полетного контроллера и ROS отличаются). После этого обработанные данные направляются в полетный контроллер, который отвечает за принятие решения: остановить летящий БПЛА или продолжать полет.
+
+#### Аппаратное решение из 4-х лазерных дальномеров
+
+Помимо лидара, было принято решение использовать статические дальномеры для удешевления системы. Нами было разработано аппаратное решение на базе Arduino Nano  и дальнедистанционных датчиков времени пролета VL53L1X . Изделие совместимо с разработанным нами креплением.
+
+Далее было написано программное обеспечение для работы с оборудованием. Было необходимо передать данные с микроконтроллера, установленного на Arduino Nano на микрокомпьютер, представить показания с датчиков в том же виде, в котором они представлены при получении данных с лидара. И после этого произвести те же действия, что и с лидаром.
+
+### Программные подсистемы «Мастер первоначальной настройки БПЛА» и «Монитор состояния БПЛА»
+
+Нами была запланирована разработка еще одной программной подсистемы, которая позволит настроить БПЛА, используя единый пользовательский веб-интерфейс, который «проведет» пользователя через весь процесс настройки. Также в данный веб-интерфейс должен быть встроен монитор текущего состояния БПЛА и редактор настроек.
+
+В первую очередь нами был сделан каркас будущего интерфейса.
+
+Также на данный момент реализован интерфейс калибровки датчиков БПЛА со встроенной анимацией для удобной работы с ним. Работу интерфейса калибровки можно увидеть на видео: [https://youtu.be/_XwKn4O9e48](https://youtu.be/_XwKn4O9e48).
```

Comment:
Wouldn't embedding the video into the article with iframe better?

Reply:
Corrected https://github.com/CopterExpress/clover/pull/309/commits/9af85ae87650246da47f9878dc2f13faf9c8a57c